hibernate-cache/src/test/java/org/infinispan/test/hibernate/cache/access/PutFromLoadValidatorUnitTest.java

```diff
@@ -273,7 +273,7 @@ private void multipleRegistrationtest(final boolean transactional) throws Except
 				if (transactional) {
 					tm.begin();
 				}
-				SharedSessionContractImplementor session = mock (SharedSessionContractImplementor.class);
+				SessionImplementor session = mock (SessionImplementor.class);
```

Comment:
Remove space between `mock` and `(` please (style should be corrected on modified lines).